### PR TITLE
Allow using Boto profile name to select credentials

### PR DESCRIPTION
**Pull request by ijjorama**
Pass setting 'profile_name' after --connection-options : makes switching between different S3 backends easier.

### CODE REVIEW

**Comment by hir12111** [comment]
Thanks for this change, it's become more flexible,
I would like to mention the first commit is actually doing the opposite,
could you squash both commits and delete the print line?

File: dosna/backends/s3.py [hunk 1]
```diff
@@ -197,6 +198,7 @@ def create_chunk(self, idx, data=None, slices=None):
             raise Exception('DataChunk `{}{}` already exists'.
                             format(self.name, idx))
         name = self._idx2name(idx)
+#        print "Name = %s" % (name)
```
Comment (EmilioPeJu) [inline dosna/backends/s3.py:201]: the print line is still there

**Review by EmilioPeJu — CHANGES_REQUESTED** [review]
Thanks for this change, it's become more flexible,
I would like to mention the first commit is actually doing the opposite,
could you squash both commits and delete the print line?
or alternatively, edit both commits to delete that line?

**Comment by ijjorama** [comment]
Will start with a clean state.

**Pull request closed without merging**